Firestore/Example/Firestore.xcodeproj/project.pbxproj

```diff
@@ -2689,6 +2885,157 @@
 			};
 			name = Release;
 		};
+		DAFF0D0321E64AC40062958F /* Debug */ = {
+			isa = XCBuildConfiguration;
+			baseConfigurationReference = 98366480BD1FD44A1FEDD982 /* Pods-macOS_example.debug.xcconfig */;
+			buildSettings = {
+				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				CLANG_ANALYZER_NONNULL = YES;
+				CLANG_ANALYZER_NUMBER_OBJECT_CONVERSION = YES_AGGRESSIVE;
+				CLANG_CXX_LANGUAGE_STANDARD = "gnu++14";
```

Comment:
C++ language standard should be C++11 without GNU extensions, which Xcode spells `"c++0x"`.

Here and throughout.